packages/models-library/src/models_library/wallets.py

```diff
@@ -0,0 +1,44 @@
+import enum
+from datetime import datetime
+from typing import TypeAlias
+
+from pydantic import BaseModel, Field, PositiveInt
+
+WalletID: TypeAlias = PositiveInt
+
+
+class WalletStatus(str, enum.Enum):
+    ACTIVE = "ACTIVE"
+    INACTIVE = "INACTIVE"
+
+
+### DB
+
+
+class WalletGetDB(BaseModel):
```

Comment:
Just wondering, why is it called WalletGet for the DB? there is no GET,POST,PUT concepts for the DB right?

Reply:
yes you are right, renamed :+1: